x-pack/plugins/apm/server/routes/services/get_services/get_sorted_and_filtered_services.ts

```diff
@@ -70,7 +71,13 @@ export async function getSortedAndFilteredServices({
       return [];
     }),
     serviceGroup
-      ? getServiceNamesFromServiceGroup(serviceGroup)
+      ? getServiceNamesFromServiceGroup({
```

Comment:
This isn't useful anymore. The intent of this function is to quickly get a list of sorted and filtered services, but as this aggregates over all the data, including raw transactions and spans, it is likely to be slower than the request that gets the actual statistics. We should remove it, I think.

Reply:
That's true, we wouldn't get the benefit of terms enum. Removing it though could result in preloaded service names being removed once we get the "final" response right?

Reply:
We probably shouldn't preload at all if a service group is selected. So, simply returning an empty array here might suffice. But I don't know how it specifically works. terms_enum doesn't make sense if a service group is selected, we shouldn't use it at all in that case (I think that's already happening today).